src/js/components/Notification/stories/typescript/Status.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { Box, Grommet, Notification, Text } from 'grommet';
+
+import { deepMerge } from 'grommet/utils';
+import { grommet, ThemeType } from 'grommet/themes';
+import { DisabledOutline } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customTheme: ThemeType = deepMerge(grommet, {
+  global: {
+    font: {
+      family: 'Arial',
+    },
+  },
+  notification: {
+    container: {
+      pad: { horizontal: '200px', vertical: '100px' },
```

Comment:
I like that we are applying different values to test types. Could we still do this, but maybe dial this back a bit to be less exaggerated?

```suggestion
      pad: { horizontal: 'large', vertical: '36px' },
```